src/routes/+page.svelte

```diff
@@ -267,24 +187,21 @@
 		transition: 0.2s;
 	}
 
-	.box-3 {
-		grid-area: box-3;
+	.chart-river-ocean {
```

Comment:
Bij eerdere classnames gebruik je graph, hier chart. Misschien een idee om er eentje te kiezen voor de consistentie?

Reply:
ik heb nu graph veranderd in chart :)